ARK Server Manager/RCONWindow.xaml.cs

```diff
@@ -506,7 +506,7 @@ public ICommand ChatPlayerCommand
 
                         CurrentInputWindowMode = InputWindowMode.ServerChatTo;
                         inputBox.Tag = player;
-                        inputTitle.Text = $"Send a chat message to : {player.SteamName}";
+                        inputTitle.Text = $"Send a chat message to : {player.SteamName ?? string.Empty}";
```

Comment:
This is an odd case.  Can we put something like (unknown) in here instead of just a blank name if it is missing?

Also, localize